apps/dotcom/client/src/tla/app/zero-polyfill.ts

```diff
@@ -182,9 +180,21 @@ export class Zero {
 		}
 		return {
 			...stuff,
-			where: (column: string, _ownerId: string) => {
+			where: (column: string, op: string, value: any) => {
+				assert(op === '=', 'Only = operator is supported')
```

Comment:
Made this explicit, just so we don't start using some operators we currently don't support.